pages/docs/tutorials/native/working-with-klib.md

```diff
@@ -0,0 +1,140 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Working with Kotlin/Native Libraries"
+description: "A look at how to work with Kotlin/Native libraries"
+authors: Hadi Hariri 
+date: 2018-01-22
+showAuthorInfo: false
+---
+
+
+In this tutorial we'll see
+
+* [What is a Kotlin/Native Libary](#what-is-kotlin/native-library)
+* [How to create a Kotlin/Native Library](#creating-a-kotlin/native-libraries)
+* [How to consume Kotlin/Native libraries](#consuming-a-kotlin/native-library)
+* [How to use and create library repositories](#using-and-creating-library-repositories)
+
+
+## What is Kotlin/Native Library
+
+The concept of a library, a collection of resources to be used by applications, is available in Kotlin/Native and it is represented
+as a file with the extension `klib`. Kotlin/Native libraries are artifacts that are only available at compile time, i.e. we do not ship these as dependencies to our application
+(as we would for instance with DLL files on the Windows platform).
+
+A `klib` file is a compressed archive in zip format with the following directory structure:
+
+File `utils.klib`
+
+```
+    targets/                    /* One or more target platforms */
+    
+        macbook/                /* When targeting macbook, this entry would appear */
```

Comment:
Seriously, "macbook"? Do I need to compile programs differently for my iMac?

Reply:
This is the compiler option. If they change it, we can update it.